pimd/pim_cmd.c

```diff
@@ -8347,11 +8349,12 @@ DEFUN (interface_ip_igmp_last_member_query_count,
 
 DEFUN (interface_no_ip_igmp_last_member_query_count,
        interface_no_ip_igmp_last_member_query_count_cmd,
-       "no ip igmp last-member-query-count",
+       "no ip igmp last-member-query-count [(1-255)]",
        NO_STR
        IP_STR
        IFACE_IGMP_STR
-       IFACE_IGMP_LAST_MEMBER_QUERY_COUNT_STR)
+       IFACE_IGMP_LAST_MEMBER_QUERY_COUNT_STR
+       IGNORED_IN_NO_STR)
 {
 	char default_robustness[2];
```

Comment:
Same here: either stringnify define or use `DESTROY` in the YANG op.

Reply:
default_robustness